core/src/main/java/org/mapstruct/BeanMapping.java

```diff
@@ -14,10 +14,51 @@
 import static org.mapstruct.NullValueCheckStrategy.ON_IMPLICIT_CONVERSION;
 
 /**
+ * // TODO should I give some nullValueMappingStrategy example or not? <br>
```

Comment:
are you going to remember all TODO's? Perhaps better (in general) to add PR comments to your own code

Reply:
Ok